integrationTests/vm/wasm/transfers/transfers_test.go

```diff
@@ -0,0 +1,36 @@
+//go:build !race
+
+package transfers
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/multiversx/mx-chain-go/integrationTests/vm/wasm"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTransfers_DuplicatedTransferValueEvents(t *testing.T) {
+	context := wasm.SetupTestContext(t)
+	defer context.Close()
+
+	err := context.DeploySC("../testdata/transferValue/output/transferValue.wasm", "")
+	require.Nil(t, err)
+
+	err = context.ExecuteSCWithValue(&context.Owner, "receive", big.NewInt(1))
+	require.Nil(t, err)
+	require.Len(t, context.LastLogs, 1)
+	require.Len(t, context.LastLogs[0].GetLogEvents(), 4)
+
+	events := context.LastLogs[0].GetLogEvents()
+
+	// There are duplicated events, to be fixed here:
```

Comment:
can we make this test fail as the current state of the `rc/v1.6.next1` and we will automatically merge this after #5936 ? The reference can remain

Reply:
Indeed. Changed.